LondonBritishLibrary/orient/BLorient751.xml

```diff
@@ -33,23 +34,160 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                          <incipit xml:lang="gez"><locus target="#2r"></locus>
+                          በስመ፡ አብ፡ <gap reason="ellipsis" resp="PRS10171Wright"></gap> ይቤ፡ ቅዱስ፡ ቀሌምንጦስ፡
+                          እምድኅረ፡ ዕርገቱ፡ ለእግዚእነ፡ ወአምላክነ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ውስተ፡ ሰማያት፡ ተፈልጡ፡ አርድእት፡ እስከ፡ አጽናፈ፡
+                          ዓለም፡ ከመ፡ ይስብኩ፡ ዜና፡ ወንጌል፡ ቅዱስ፡ ወይሚጥዎሙ፡ ለአሕዛብ፡ ውስተ፡ አሚን፡ ወአእምሮ፡ ወያጥምቅዎሙ፡ በቅድስት፡
+                          ጥምቀት፡ ዘሐዲስ፡ ልደት። ወበእንተዝ፡ ፈቀዱ፡ ይኅሥሡ፡ ውስተ፡ በሓውርት። ወእምዝ፡ ነሥአኒ፡ ሊተ፡ ስምዖን፡ ጴጥሮስ፡ ወረሰየኒ፡
+                          ረድአ፡ ዚአሁ፡ እስመ፡ አመንኩ፡ ቦቱ፡ ወበዘፈነዎ። ወጠየቁ፡ ከመ፡ ውእቱ፡ ርእሶሙ፡ ለሐዋርያት፡ በእንተ፡ ቃል፡ ዘይቤ፡ እግዚእነ፡
+                          ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ለረድኡ፡ ጴጥሮስ፡ አንተ፡ ኰኵሕ፡ ወዲበ፡ ዛቲ፡ ኰኵሕ፡ አሐንጻ፡ ለቤተ፡ ክርስቲያንየ፡ ወኢይክልዋ፡
+                          አናቅጸ፡ ሲኦል፡ አማስኖታ፡ ወኢይኄይልዋ። ወእሁበከ፡ ለከ፡ መራኁተ፡ መንግሥተ፡ ሰማያት። ወእምድኅረ፡ ጕንዱይ፡ ዕለት፡ ነሥአ፡ ክልኤ፡
+                          አኃወ፡ ወረሰዮሙ፡ አርድእተ፡ ዚአሁ። ወእምዝ፡ ሶበ፡ ነበርኩ፡ ምስሌሁ፡ ዓሠርተ፡ ዓመተ፡ ነሥአኒ፡ ወአብጽሐኒ፡ ኀበ፡ አቡየ፡ ወእምየ፡ እንተ፡
+                          ስማ፡ ምጥሮርያ፤ ወከሠተ፡ ሊተ፡ ምሥጢረ፡ ዘመሀሮ፡ እግዚእነ፡ ኢየሱስ፡ ክርስቶስ፡ በደብረ፡ ዘይት፤ ወውእተ፡ ጊዜ፡ ነበሩ፡ ሐዋርያት፡
+                          ወኵሎሙ፡ መሃይምናን፡ ውስተ፡ ዐቢይ፡ ምንዳቤ፡ ወስደት፡ እምኀበ፡ ዕልዋን፡ ወከሓድያን፡ አይሁድ። እስመ፡ አይሁድ፡ ነበሩ፡ እንዘ፡
+                          ይቀትሉ፡ ኵሎ፡ ዘረከቡ፡ መሃይምናን። ወእምዝ፡ ሖረ፡ ኄር፡ ጴጥሮስ፡ ወቦአ፡ ውስተ፡ አሐቲ፡ ሀገር፡ ወአነሂ፡ ምስሌሁ። ወበህየ፡ ረከበኒ፡
+                          ዐቢይ፡ ምንዳቤ፡ እምተጋዕዞቶሙ፡ ለከሓድያን፡ አይሁድ። ወተስእሎቶሙ፡ በእንተ፡ ልደታ፡ ለማርያም፡ ቅድስት፡ እስመ፡ ይስእሉ፡ በእንቲአሃ፡
+                          እስመ፡ ኢኮነት፡ እምትዝምደ፡ ይሁደ፡ ከመ፡ ያብጥሉ፡ ምጽአቶ፡ ለእግዚእነ፡ ወመድኃኒነ፡ ኢየሱስ፡ ክርስቶስ፡ ውስተ፡ ዓለም። እስመ፡
+                          እሙንቱ፡ ነበሩ፡ እንሸ፡ ይሁቡ፡ ንዋየ፡ ብዙኃ፡ ለሰብአ፡ ጽርዕ፡ ወለሰብአ፡ ሮሜ፡ ከመ፡ ይርድእዎሙ፡ በእንተ፡ ጥፍአቶሙ፡
+                          ለመሃይምናን፡ ወከልእዎሙ፡ ለሐዋርያት፡ አንብቦ፡ ኦሪት፡ ከመ፡ ያእምሩ፡ ቀዳሚ፡ ፍጥረተ፡ ዓለም። ወሶበ፡ ርኢኩ፡ ዘንተ፡ ኵሎ፡
+                          ሕማመ፡ ወምንዳቤ፡ ዘመጽአ፡ ላዕሌነ፡ እምኀበ፡ አይሁድ፡ ከሓድያን፡ ኀሠሥኩ፡ አነ፡ ኀበ፡ መምህርየ፡ ስምዖን፡ ጴጥሮስ። ወተስእልክዎ፡
+                          ከመ፡ ያለብወኒ፡ ወይንግረኒ፡ በከመ፡ ኮነ፡ ቀዳሚ፡ ፍጥረተ፡ ዓለም። እስመ፡ ውእቱ፡ አእመረ፡ ኵሎ፡ ምሥጢራተ፡ እምኀበ፡ እግዚእነ፡
+                          ኢየሱሱ፡ ክርስቶስ፡ ወአንሰ፡ አአምር፡ በነገረ፡ ጽርዕ፡ ወበመጻሕፍቲሆሙ፡ ወኵሎ፡ ምሥጢራቲሆሙ፡ ወጥበባቲሆሙ። ወነገርክዎ፡ ኵሎ፡
+                          ለመምህርየ፡ ዘመጽአ፡ ላዕሌየ፡ ቂመ፡ ወቅንዓተ፡ በእንተ፡ ዘጸልኡኒ፡ ወተጋዓዙኒ፡ አይሁድ፡ በእንተ፡ ማርያም፡ ቅድስት፡ እስመ፡ አልብየ፡
+                          አእምሮ፡ በኦሪት፡ ወበነቢያት፡ ወብዙኃን፡ ይጽዕሉኒ፡ በእንተ፡ ፍጥረቱ፡ ለአዳም፡ አቡነ፡ ወፈድፋደ፡ ይጸርፉ፡ ላዕለ፡ እግዚእትነ፡ ማርያም፡
+                          ቅድስት፡ ወኢረክቡ፡ ምክንያተ፡ በዘአአትት፡ እከዮሙ፡ ወርኩሰ፡ ቃሎሙ። ወሶበ፡ ነገርክዎ፡ ዘንተ፡ ኵሎ፡ ለመምህርየ፡ ተከዘ፡ ፈድፋደ፡ ወቦአ፡
+                          ላዕሌሁ፡ ቅንዓት፡ ወይቤለኒ፡ አነ፡ እከሥት፡ ለከ፡ ኦወልድ፡ ኵሎ፡ ዘሰአልከኒ፡ ወአሌብወከ፡ ዘእምፍጥረተ፡ ዓለም፡ ቀዳሚ፡ ወአጤይቀከ፡ በእንተ፡
+                          ቅድስት፡ ማርያም፡ እሙ፡ ለብርሃን፡ ከመ፡ ይእቲ፡ ጥይቅት፡ ዘእንበለ፡ አስትቶ፡ እምትዝምደ፡ ይሁደ፡ ወልደ፡ ያዕቆብ። ወዓዲ፡ እዜንወከ፡ ዘከመ፡
+                          ወድቀ፡ ሰይጣን፡ እምሰማይ።
+                        </incipit>
+
+                        <msItem xml:id="ms_i1.1">
+                          <locus from="2v"></locus>
+                          <title type="complete" xml:lang="en">Account of the creation, and a resume of Scripture
```

Comment:
Here and everywhere: ideally, one should edit the work entry, LIT1957Mashaf, introducing the chapters with IDs (noting that the edition comes from this MS), and add derived IDs to the titles of msItems here

(the incipits ideally should also be copied there)

Reply:
I plan to edit the work entry, just want to encode all manuscripts containing this work and check contributions on the work next time in the library to be consistent. When it is done, I can add IDs to the manuscript description.

Reply:
እንሸ or እንዘ?